src/presentation/ui/component/runtimes/phpRuntimeTabContent.templ

```diff
@@ -0,0 +1,268 @@
+package componentRuntimes
+
+import (
+	"github.com/speedianet/os/src/domain/entity"
+	"github.com/speedianet/os/src/domain/valueObject"
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+	presenterDto "github.com/speedianet/os/src/presentation/ui/presenter/dto"
+	"strconv"
+)
+
+script PhpRuntimeTabContentLocalState() {
+	document.addEventListener('alpine:init', () => {
+		Alpine.data('php', () => ({
+			// Primary states
+			selectedVersion: '',
+
+			// Auxiliary states
+			selectedConfigType: 'modules',
+			setSelectedConfigType(configType) {
+				this.selectedConfigType = configType;
+			},
+			updateModuleStatus(vhostHostname, version, moduleName, moduleStatus) {
+				htmx.ajax(
+					'PUT',
+					'/api/v1/runtime/php/' + vhostHostname + '/',
```

Comment:
I added `vhostHostname` as a state in Alpine.js. I'd like to do the same for the PHP version as well. Although it is already a state, I only use it to create the warning modal. Otherwise, I still inject the value through function param. This is because when the service is installed, it provides me with the selected `vhostHostname`, but it may or may not have a mapping created for PHP, which directly affects whether I'll have access to the version or not. This lack of control prevents me from adding the PHP version directly into Alpine.js state, as I'm now doing with `vhostHostname`.